CHANGELOG.md

```diff
@@ -2,6 +2,47 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.17 - April 14, 2021
+## Breaking changes ⚠️
+* [Annotation plugin] Rename annotation classes, the rules are as follows and applied for Annotations/Options/Managers ([#227](https://github.com/mapbox/mapbox-maps-android/pull/227))
+    - Symbol -> PointAnnotation
+    - Circle -> CircleAnnotation
+    - Line -> PolylineAnnotation
+    - Fill -> PolygonAnnotation
+* mapboxMap.queryRenderedFeatures will return a new data class QueriedFeature which will contain additional properties ([#247](https://github.com/mapbox/mapbox-maps-android/pull/247)): 
+    - source (id of the source)
+    - sourceLayer (id of the source's layer)
+    - state (feature's state) 
+* Deprecate MapObserver, MapSnapshotterObserver interfaces and align sync method naming, Map#isMapLoaded, StyleManager#isStyleLoaded
+* Remove old map#drag API and the AnimationOptions API
```

Comment:
@kiryldz thoughts on this entry? Is this API public-facing?

Reply:
Yes, it is public, so let's leave it.